node/node.go

```diff
@@ -119,7 +121,11 @@ func NewNode(config cfg.Config, privValidator *types.PrivValidator, clientCreato
 	consensusReactor := consensus.NewConsensusReactor(consensusState, fastSync)
 
 	// Make p2p network switch
-	sw := p2p.NewSwitch(config.GetConfig("p2p"))
```

Comment:
hmm okay, well I rebuilt this functionality (because it doesn't exist in viper) in the few added lines right below, if there is no functional use for this code of loading in p2p from a config instead of just initializing a new config for go-p2p, then I reckon it makes sense to remove or comment out this code.